test/ALK.js

```diff
@@ -119,16 +120,15 @@ describe('ALK', done => {
     parser.parse(heading_nineC)
   })
 
-/*
-  it('Doesn\'t choke on empty sentences', done => {
-    const parser = new Parser
-    parser
-    .parse('$STALK,9C,,,*3B')
-    .then(result => {
-      should.equal(result, null)
-      done()
-    })
-    .catch(e => done(e))
+  it('Doesn\'t choke on empty 0x9C sentences', () => {
```

Comment:
Re-enabled this test and added a similar one for 0x84. Changed sub-hooks to make them pass.